README.md

```diff
@@ -4,13 +4,20 @@
 
 # Arclight
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/arclight`. To experiment with that code, run `bin/console` for an interactive prompt.
+A Rails engine supporting discovery of archival materials described in the [Encoded Archival Description](http://eadiva.com/2/) (EAD) format.
```

Comment:
I'd be less specific here but add a note (perhaps in the MVP description) that initial development targeted ingest of EAD 2002.